models/streamitem_test.go

```diff
@@ -0,0 +1,146 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/bwmarrin/snowflake"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStreamItemStore(t *testing.T) {
+	tx := TestDB.Begin()
+	defer tx.Rollback()
+
+	store := NewStreamItemStore(tx)
```

Comment:
If you're using a transaction (like all tests should, just not all of them do), you have to start a new transaction for each test case, because the first failure will abort it.